core-concepts/connecting-to-databases/querying-firestore.md

```diff
@@ -0,0 +1,114 @@
+# Querying Firestore
+
+The Firestore plugin can connect to Firestore an execute the set of officially supported ways of querying. This document describes connecting Appsmith to a Firestore instance and applying the various operations on a Firestore instance.
+
+## Configuring a Firestore Datasource
+
+The Firestore plugin requires three pieces of information to connect to Firestore. Here's the form in Appsmith when creating a new Firestore datasource:
+
+![Click to expand](../../.gitbook/assets/firestore-datasource-form.png)
+
+These three details are available from the Firebase Console. Let's fetch them in turn.
+
+Go to "Project Settings" in your Firebase console and ensure you're in the "General" tab. Here, you'll find a "Project ID" entry with a value. Copy this value into the "Project ID" field in the datasource configuration.
```

Comment:
Can we enumerate these steps in an ordered list? Makes it easier to understand & grok in a quick glance as to the number of steps required to connect to a Firestore instance.